src/jit/lsrabuild.cpp

```diff
@@ -2645,6 +2645,12 @@ int LinearScan::BuildDelayFreeUses(GenTree* node, regMaskTP candidates)
         setDelayFree(use);
         return 1;
     }
+    else if (node->OperIsHWIntrinsic())
```

Comment:
Nit: (And this applies to the existing `id` below) this doesn't need to be `else if`, as the clause above returns.

Reply:
I'll fix up after the tests finish, that way I don't need to requeue the outer-loop jobs again 😄